kudo-operator/params.yaml

```diff
@@ -0,0 +1,121 @@
+# Service Accounts and RBAC
+operatorServiceAccountName:
+  description: "Service Account for Spark Operator"
+  default: "spark-operator-service-account"
+  displayName: "Operator Service Account"
+
+createOperatorServiceAccount:
+  description: "Automatically create Service Account for Spark Operator"
+  default: true
```

Comment:
It follows the way it's done in Helm charts with the only difference, that SA names are specified here. The default settings for all the auto-create resources is to make everything work out of the box with dedicated service accounts and RBACs.

The logic is the following:
- if users set `createOperatorServiceAccount: false` then they should create SA on their own and provide it via `operatorServiceAccount`
- if users want to use different Service Account Name, they just provide `operatorServiceAccountName: XYZ` and it will be created for them
- users can set `createOperatorServiceAccount: false` and `operatorServiceAccountName: "default"` to use `default` service account

The reasoning here is that a secure deployment should always use non-default service accounts for the finer-grained resource and security management. On the other hand, using `createOperatorServiceAccount: false` and `operatorServiceAccountName: "default"` also makes sense due to simplicity but only if users can still do one-click install without additional RBAC configuration. WDYT?

Reply:
That sounds reasonable to me. My question was mostly around how much do we handle things behind the scenes for users vs forcing them to conscientiously do it themselves. That said, one-click install is always the optimal path to pursue for UX.